llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -547,15 +555,36 @@ static string_vector saveDeviceImageProperty(
 
     {
       Module *M = ResultModules[I].get();
+      bool HasIndirectlyCalledAssert = false;
+      std::vector<llvm::Function *> Kernels;
       for (auto &F : M->functions()) {
         // TODO: handle SYCL_EXTERNAL functions for dynamic linkage.
         // TODO: handle function pointers.
```

Comment:
This TODO can be now removed

Reply:
Not sure that it's right to remove it - we haven't implemented this yet.